images/intellij/Dockerfile.ubuntu

```diff
@@ -3,9 +3,10 @@ FROM codercom/enterprise-multieditor:ubuntu
 # Run everything as root
 USER root
 
-# Install intellij.
+# Install intellij & java.
 RUN mkdir -p /opt/idea
 RUN curl -L "https://download.jetbrains.com/product?code=IIC&latest&distribution=linux" | tar -C /opt/idea --strip-components 1 -xzvf -
+RUN sudo apt install default-jre -y
```

Comment:
This is already running as root, so just apt-get install should be fine. I suggest using --no-install-recommends and specifying the version, i.e. `apt-get install -y --no-install-recommends openjdk-11-jdk`

The JDK is important (rather than JRE), since IntelliJ is for building Java applications, not just running them.

I think IntelliJ also ships a JDK so I'm not sure if that's what was being used before, do you know if that's the case?

Reply:
I'm not entirely sure. All I know is that `java` wasn't installed on the command line. I also ran `which java`, which returned null.

Reply:
```suggestion
RUN apt-get install -y --no-install-recommends openjdk-11-jdk
```